### DISCUSSION TITLE
PYTD Stopped Working

### DISCUSSION THREAD

**Discussion by ghost** [sql-bi/Bravo#762] [Q&A]
HI All,

I need help.

I have used Bravo in my Power BI sheet for a while on 3 different measures. All worked ok with the PYTD calculations and tracked everything ok. As in showing the Last years total value up to today date. 08/01/23 = £100,000 etc.
My YTD calculations for this year 2024 are working right.

Now however, since adding in more time intelligence to other measures, my PYTD calculations is now showing the total value invoiced for 2023 £10,000,000 etc, instead of showing the invoiced amount up to todays date. 

Anyone know why these calculation would stop working suddenly once adding in more measures to bravo?

**Comment by albertospelta** [comment]
Hi, in addition to creating new time intelligence measures, did you also refresh data or add new table to the model? 
Does your date table contain any rows where `'Date'[DateWithTransactions] = FALSE` ? 
Note that `DateWithTransactions` is a column automatically created by Bravo and used in PYTD time intelligence measures.
